config.js.template

```diff
@@ -10,6 +10,9 @@ module.exports = {
   network: 'mainnet',
   server: 'https://node2.mainnet.hathor.network/v1a/',
 
+  // Tx Mining Service
```

Comment:
I think we can say that it's an optional config and if not set it will use the default from the lib. What do you think?